### DISCUSSION TITLE
Project Roadmaps and Mobile support 🤩

### DISCUSSION THREAD

**Discussion by nyqvistatwork** [community/events#64] [Community]
Finally ! We getting **Roadmaps** to Github Projects and **support for Projects on the mobile app** ❤

**Comment by jricciardi** [comment]
So exciting!
![image](https://media.giphy.com/media/huJmPXfeir5JlpPAx0/giphy.gif)

**Comment by ghostinhershell** [comment]
![image](https://media1.giphy.com/media/1hc7YW5RTgz8EmCm5I/giphy.gif?cid=ecf05e47rvf8nw3nbf7iznexd6laebq6scje8r0am0gie6f7&rid=giphy.gif&ct=g)
